Zend/zend_operators.h

```diff
@@ -25,6 +25,7 @@
 #include <math.h>
 #include <assert.h>
 #include <stddef.h>
+#include <ctype.h>
```

Comment:
if `<ctype.h>` needed here? The conversion should be locale independent, so probably not.

Reply:
No, can be removed, this is leftover code.